### DISCUSSION TITLE
Error "Can not find element in DOM" in Vue.js quasar

### DISCUSSION THREAD

**Discussion by amirsafaricg** [tradingview/lightweight-charts#967] [Q&A]
hey guys this is my code and this is the error I get when I run it 
I can't implement the chart's on my chart's (Library : https://quasar.dev/ )
codepen link : 
https://codepen.io/amirsafaricg/pen/VwMGVrY

this is the error I get : 
"Assertion failed : cannot find element in DOM with id = container"

**Comment by timocov** [comment]
See my answer here https://github.com/tradingview/lightweight-charts/discussions/964#discussioncomment-1946235
